docker/Dockerfile

```diff
@@ -41,34 +38,30 @@ ENV PIP_USE_PEP517=1
 FROM python-base AS pyproject-builder
 
 # Install dependencies
-RUN \
-  --mount=type=cache,target=/var/cache/apt,sharing=locked \
-  --mount=type=cache,target=/var/lib/apt,sharing=locked \
-  apt-get update \
+RUN apt-get update \
   && apt-get install -y \
     --no-install-recommends \
     build-essential=12.9 \
     gcc=4:10.2.* \
-    python3-dev=3.9.*
+    python3-dev>=${PYTHON_VERSION}
```

Comment:
last time I checked there was no version of the python 3.10 headers available for the slim image, has this changed?